railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -354,7 +354,7 @@ class TwoMigration < ActiveRecord::Migration::Current
           use_postgresql(multi_db: true)
           generate_models_for_animals
 
-          rails "db:create:animals", "db:migrate:animals", "db:create:primary", "db:migrate:primary", "db:schema:dump"
+          rails "db:create:animals", "db:create:primary", "db:migrate:animals", "db:migrate:primary", "db:schema:dump"
```

Comment:
@eileencodes Fixed. Didn't see any other issues that weren't failing on master.

Reply:
(Assuming this is okay behavior. Annotations cannot be dumped unless all databases have been created.)